cmd/opm/alpha/bundle/generate.go

```diff
@@ -44,11 +45,18 @@ func newBundleGenerateCmd() *cobra.Command {
 	bundleGenerateCmd.Flags().StringVarP(&outputDirArgs, "output-dir", "u", "",
 		"Optional output directory for operator manifests")
 
+	bundleGenerateCmd.Flags().BoolVarP(&synthesizeCSVArgs, "synthesize-csv","y",false,
+		"Synthesize ClusterServiceVersion from a series of kubernetes manifests in a registry+v2 bundle")
+
+	bundleGenerateCmd.Flags().BoolVarP(&validateCSVArgs, "validate-csv", "v", false,
+		"Validating ClusterServiceVersion present in the (output) bundle directory")
```

Comment:
We should keep consistent tense here, all the others are framed as verb whereas this is framed as present continuous tense, i.e `Validating` instead of `Validate`